lib/polyfill/mediasource.js

```diff
@@ -214,7 +215,8 @@ shaka.polyfill.MediaSource = class {
           ManagedMediaSource.isTypeSupported.bind(ManagedMediaSource);
 
       window.ManagedMediaSource.isTypeSupported = (mimeType) => {
-        const actualCodec = shaka.util.MimeUtils.getCodecBase(mimeType);
+        const codecs = shaka.util.MimeUtils.getCodecs(mimeType);
```

Comment:
Code change looks good to me.  Only the tests need improvement AFAICT.  Thank you for contributing!

Reply:
Also, please take a look at the linter errors.  You can run `python3 build/check.py` to see them locally before your next push.